apps/todos/src/app/todos/todos.component.html

```diff
@@ -1,9 +1,10 @@
-<h1 class="todo__h1" data-testId="app-heading">todos</h1>
+<h1 class="todo__h1" data-testId="app-heading">{{ appTitle }}</h1>
```

Comment:
Vielleicht auch etwas kürzer?
Ich habe das camelCase in testID noch nicht verstanden.
```suggestion
<h1 class="todo__h1" data-test="app-heading">{{ appTitle }}</h1>
```

Reply:
Ja stimmt, camelcase hab ich mir irgendwann komischerweise angewöhnt, aber ist natürlich eigentlich quatsch. Ich nehm aber normalerweise testid, weil es der Standard bei @testing-library/cypress ist. Ansonsten muss man es umkonfigurieren https://testing-library.com/docs/queries/bytestid#overriding-data-testid